ush/templates/FV3LAM_wflow.xml

```diff
@@ -1044,6 +1087,7 @@ MODULES_RUN_TASK_FP script.
   </metatask>
 {% endif %}
 
+{% if not do_ensemble %}
```

Comment:
The ensemble part does not need the python graphics, clean and archive tasks?

There should be better way to turn off python graphics but not through "not do_ensemble".

Reply:
For ensemble, the graphics/archive tasks are different. I am turning them off for ensemble for now to keep it simple and will add some switches later.

Reply:
If this is a very temporary things, suggest you keep it in your rrfs_ens branch and commit things back to RRFS_dev1 when it is ready.

Reply:
Added these tasks back for ensemble.